pkg/testgridanalysis/testgridanalysisapi/types.go

```diff
@@ -69,3 +72,7 @@ const (
 	Success = "Success"
 	Failure = "Failure"
 )
+
+var (
+	OperatorConditionsTestCaseName = regexp.MustCompile("operator (install|conditions) (?P<operator>.*)")
```

Comment:
i don't see any failures matching "operator conditions", is this a pending test case name change?
https://search.ci.openshift.org/?search=operator+conditions&maxAge=48h&context=1&type=bug%2Bjunit&name=&maxMatches=5&maxBytes=20971520&groupBy=job

Reply:
i see https://github.com/openshift/release/pull/12298 now.